dockers/docker-orchagent/orchagent.sh

```diff
@@ -51,7 +52,7 @@ elif [ "$platform" == "mellanox" ]; then
 elif [ "$platform" == "innovium" ]; then
     ORCHAGENT_ARGS+="-m $MAC_ADDRESS"
 else
-    MAC_ADDRESS=`sonic-cfggen -d -v 'DEVICE_METADATA.localhost.mac'`
+    # Should we use the fallback MAC in case it is not found in Device.Metadata
```

Comment:
had a question here? it is not clear to me why we call again into sonic-cfggen.

Reply:
The change looks good to me. Then I see only Mellanox is different, all others shared the same logic.

---
In reply to: [452389446](https://github.com/Azure/sonic-buildimage/pull/4936#discussion_r452389446) [](ancestors = 452389446)